src/stores/osimRuntime.ts

```diff
@@ -23,29 +23,30 @@ export const osimRuntimeStatus = computed<OsimRuntimeStatus>(() => {
 });
 
 const OsimRuntime = z.object({
+  env: z.string().default(''),
   backends: z.object({
     osidb: z.string(),
     osidbAuth: z.string().default('kerberos'),
     bugzilla: z.string(),
+    jira: z.string(),
+  }),
+  osimVersion: z.object({
+    rev: z.string(),
+    tag: z.string(),
+    timestamp: z.string(),
+    dirty: z.boolean(),
   }),
-  osimVersion: z.string(),
   error: z.string().default(''),
 });
 type OsimRuntime = z.infer<typeof OsimRuntime>;
 
 const runtime = ref<OsimRuntime>({
-  backends: {
-    osidb: '',
-    osidbAuth: '',
-    bugzilla: ''
-  },
-  osimVersion: '',
+  env: '',
+  backends: { osidb: '', osidbAuth: '', bugzilla: '', jira: '' },
+  osimVersion: { rev: '', tag: '', timestamp: '', dirty: true },
   error: '',
 });
-export const osimRuntime = computed<OsimRuntime>(() => {
-  return runtime.value;
-});
-
+export const osimRuntime = readonly(runtime.value);
```

Comment:
Also changing this to readonly creates two problems which I've encountered:

1. when changing osimRuntime to readonly, access to it changes from `osimRuntime.value.xxx` to just `osimRuntime.xxx`, there are several occurrences of that throughout the code
2. changing this to readonly produces osimRuntime with all values empty and the fetch from the runtime json file no longer mutates this  so whenever I access the osimRuntime in some different component I get empty strings I suppose this is not intended?

Reply:
It is not intended, I'd provided a code suggestion that contained non-functioning code that did not account for the API difference between `computed` and `readonly`. Thank you for your attentiveness, Jakub 👊